core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1417,10 +1417,13 @@ class DAGScheduler(
       case exceptionFailure: ExceptionFailure =>
         // Nothing left to do, already handled above for accumulator updates.
 
+      case _: TaskKilled =>
```

Comment:
nit: combine this with the `ExceptionFailure` case.

Reply:
will do.